### ISSUE TITLE
I cannot use from-json.

### ISSUE THREAD

**Issue by veer66** [metosin/jsonista#26]
I cannot use from-json as shown in [the document](https://metosin.github.io/jsonista/jsonista.core.html). Moreover, I cannot find from-json in this repository except in [the comment](https://github.com/metosin/jsonista/blob/7facbc2af01a92469ed787ee172fa02de137a0e9/src/clj/jsonista/core.clj#L23).

Should I use read-value instead? 

![ภาพ](https://user-images.githubusercontent.com/370688/75217231-f8f36d80-57c8-11ea-9f98-5049089b06e2.png)

**Comment by mattiuusitalo** [comment]
That's right: the documentation was incorrect at that point. The documentation has been corrected a while ago so closing this issue.
